files/en-us/glossary/index.html

```diff
@@ -10,26 +10,26 @@
   - Landing
   - Terminology
 ---
-<p class="summary" style="padding-top: 0; border-top-width: 0;"><span class="seoSummary">Web technologies contain long lists of jargon and abbreviations that are used in documentation and coding. This glossary provides definitions of words and abbreviations you need to know to successfully understand and build for the web.</span></p>
+<p>Web technologies contain long lists of jargon and abbreviations that are used in documentation and coding. This glossary provides definitions of words and abbreviations you need to know to successfully understand and build for the web.</p>
+
+<p>Glossary terms can be selected from the sidebar.</p>
```

Comment:
On mobile, there are no sidebars, it is below the main text. How could we formulate this in a more neutral way?

Reply:
Good point - from the menu:

```suggestion
<p>Glossary terms can be selected from the menu.</p>
```